lib/enigma.rb

```diff
@@ -13,15 +13,15 @@ def initialize
 
   def encrypt(message, key = @shifts.padded_random_number, date = @shifts.todays_date)
     {
-      :encryption => @encryption.encrypt_message(message, @shifts.shifts(key, date)),
+      :encryption => @encryption.crypt_message(message, @shifts.shifts(key, date)),
```

Comment:
Changed method name to crypt_method to apply to decryption and encryption classes. Good example of polymorphism!